data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1066,6 +1066,99 @@ ship "Centipede" "Centipede (Teeneep Jump)"
 
 
 
+ship "Frugal Defense Cannon"
+	attributes
+		"heat dissipation" 1
+		"hull threshold" 200
+		# Once this loses 50 hull ((400 - 200) / 4 hull = 50), it will stop being frugal.
+		# Since the gun does 0.5 hull damage per second, this will take exactly 100 seconds.
+		# The total time it takes to make this switch is also important to be sure that the gun is properly in position, too.
+		"hull" 400
+		"mass" 25000
+		"drag" 25
+		"automaton" 1
+		"landing speed" 1
+	outfits
+		"Timer Weapon Rangeless"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+		"Defense Cannon"
+	turret 0 0
+	turret -80 20
+	turret -70 -10
+	turret -60 20
+	turret -50 -10
+	turret -40 -20
+	turret -30 10
+	turret -20 -20
+	turret -10 0
+	turret 0 10
+	turret 0 20
+	turret 10 0
+	turret 20 -30
+	turret 30 20
+	turret 40 0
+	turret 50 10
+	turret 60 -20
+	turret 70 10
+	turret 80 0
+
+outfit "Defense Cannon"
+	category "Turrets"
+	thumbnail "outfit/unknown"
+	"turret mounts" -1
+	cost 100000000000
+	weapon
+		ammo "Timer Weapon Rangeless" 0
+		sprite "projectile/pulse"
+		sound "pulse"
+		"hit effect" "pulse impact"
+		"inaccuracy" 1
+		"turret turn" 2.3
+		"velocity" 10
+		"lifetime" 150
+		"reload" 10
+		"shield damage" 45
+		"hull damage" 34
+
+outfit "Timer Weapon Rangeless"
+	plural "Timer Weapon Rangeless"
+	category "Turrets"
+	thumbnail "outfit/unknown"
+	"turret mounts" -1
+	weapon
+		"velocity override" 10000
+		"range override" 10000
+		"lifetime" 0
+		"reload" 60
+		"firing hull" .5
+		"turret turn" 10
+
+hazard "Keep Cannon Close"
+	"constant strength"
+	"range" 1000
+	weapon
+		"gravitational"
+		# The higher the mass, the more effect (so anything other than the cannon ship should be +- ignored).
```

Comment:
I don't think this works like this.

According to https://github.com/endless-sky/endless-sky/wiki/CreatingOutfits :

`"gravitational": this tag (just by itself, no value following it) causes all ships impacted by this weapon to receive the same amount of hit force, as opposed to hit force decreasing in its effectiveness against heavier ships. (v. 0.9.13)`

This may be a non-starter without an alternate solution here?

Reply:
In that case mb best it's not grav
The mass-force ratio needs to be tested tho

Reply:
gravitation causes all ships to experience the same amount of *acceleration* due to hit force, the description is wrong in trying to be casual and easy-to-understand.